src/api/v1/employees/routes/update.ts

```diff
@@ -0,0 +1,50 @@
+import 'reflect-metadata'
+import { FastifyReply, FastifyRequest } from 'fastify'
+import * as jf from 'joiful'
+
+interface Body {
+  id: string
+  firstName: string
+  lastName: string
+  title: string
+  department: string
+  salary: number
+  secLevel: number
+  permanent: boolean
+  projects: string[]
+}
```

Comment:
Consistent spacing pls
```suggestion
}

```